lib/streaming-pipeline.js

```diff
@@ -150,16 +154,13 @@ module.exports = class StreamingPipeline extends Writable {
         }
     }
 
-    get parameterCount() {
-        return this._userFunction.$arity;
-    }
-
-    _wireInputs(inputCount, transformers) {
+    _wireInputs(transformers) {
```

Comment:
do argument transformers or `$argumentTransformers` make sense anymore? For streaming functions I'd be inclined to make messages the default and let the user get the payload themself.

Reply:
I'd rather move this discussion to a separate issue.

Reply:
See https://github.com/projectriff/node-function-invoker/issues/122